### ISSUE TITLE
Permissions not being requested in fdroid version.

### ISSUE THREAD

**Issue by Innomen** [slavabarkov/tidy#17]
moto, android 11

**Comment by Unacceptium** [comment]
true, i did a fresh install this moment, these screenshots show that this issue prevents the app from usage.
**the permission does not matter** and the app still cannot load
![Screenshot_20231228_201842](https://github.com/slavabarkov/tidy/assets/145642963/0da3c738-140b-4fe5-9413-05f43032e2f5)
![Screenshot_20231228_201852](https://github.com/slavabarkov/tidy/assets/145642963/9a1305ea-e16c-4b39-989b-c4381785dfa7)

**Comment by inson1** [comment]
same here - wasnt requested and when I set it, the app still didnt load, android 11 too, xiaomi

**Comment by gracetyy** [comment]
Same issue here. (Lenovo K13 Pro, Android 12)

**Comment by harcesz** [comment]
Same, android 11
